scripts/init

```diff
@@ -1,46 +1,42 @@
 #!/bin/bash
 
 register_node() {
-  args="--batch --no-sync"
+  args="init register"
```

Comment:
` /opt/wallarm/register-node` is a shell script and it already contains `init` argument, so I think it should be like this (with this one you provide arguments for register job)
```suggestion
  args="job:register"
```